db/migrate/20240229235448_add_is_zip_to_supporting_documents.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsZipToSupportingDocuments < ActiveRecord::Migration[7.1]
+  def change
+    add_column :supporting_documents, :is_zip, :boolean
```

Comment:
@bkmaibach - I think we should remove this one.  I don't think we should have the zip file against the supporting docs. 
The generated submission maybe, but not this.  It also should never be a zip.